src/containers/shared/components/Transaction/EscrowFinish/Description.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+import { useTranslation, Trans } from 'react-i18next'
+import { Link } from 'react-router-dom'
+import { normalizeAmount, findNode } from '../../../transactionUtils'
+import Account from '../../Account'
+import {
+  TransactionDescriptionComponent,
+  TransactionDescriptionProps,
+} from '../types'
+
+const Description: TransactionDescriptionComponent = (
+  props: TransactionDescriptionProps,
+) => {
+  const { t, i18n } = useTranslation()
+  const language = i18n.resolvedLanguage
+  const { data } = props
+  const deleted: any = findNode(data, 'DeletedNode', 'Escrow')
+
+  if (!deleted) {
+    return null
+  }
+  return (
+    <>
+      <div key="line1">
+        {t('escrow_completion_desc')} <Account account={data.tx.Account} />
+      </div>
+      <div key="line2">
+        <Trans i18nKey="escrow_completion_desc_2">
+          The escrowed amount of
+          <b>
+            {normalizeAmount(deleted.FinalFields.Amount, language)}
+            <small>XRP</small>
+          </b>
+          was delivered to
+          <Account account={deleted.FinalFields.Destination} />
+        </Trans>
+        {deleted.FinalFields.Destination === data.tx.Account && (
+          <span>
```

Comment:
The code formatting here is very strange.

Reply:
how so?

Reply:
Line 40 has a paren and its matching is on line 48 not on a new line.  Just no realized it is because of the text and they arent being used as code